### DISCUSSION TITLE
Batching loader requests

### DISCUSSION THREAD

**Discussion by JClackett** [remix-run/remix#4940] [General]
Just an open question, would remix benefit from batching all loader requests into one http call? This would be after the initial document load, on client side transitions

**Comment by sergiodxa** [comment]
That would make caching individual loaders impossible because the batches request could include one, two or many more loaders and each one would be a new different http cache key.

**Reply by JClackett** [reply]
Yeah I guess it would pick the outer/lowest leafs cache config, doesn't it already do this on initial document request anyway, (at least for the html)?
